test/Interpreter/SDK/objc_getClass.swift

```diff
@@ -1,15 +1,15 @@
 // RUN: %empty-directory(%t)
 
-// RUN: %target-build-swift-dylib(%t/libresilient_struct.%target-dylib-extension) -Xfrontend -enable-resilience -Xfrontend -enable-class-resilience %S/../../Inputs/resilient_struct.swift -emit-module -emit-module-path %t/resilient_struct.swiftmodule -module-name resilient_struct
-// RUN: %target-codesign %t/libresilient_struct.%target-dylib-extension
+// RUN: %target-build-swift-dylib(%t/%{target-shared-library-prefix}resilient_struct%{target-shared-library-suffix}) -Xfrontend -enable-resilience -Xfrontend -enable-class-resilience %S/../../Inputs/resilient_struct.swift -emit-module -emit-module-path %t/resilient_struct.swiftmodule -module-name resilient_struct
```

Comment:
This pattern is very verbose. Maybe we should make it another regex-based substitution? `%target-shared-library(resilient_struct)`.

Reply:
I tried to do that, but failed at that, but I thought it would be possible to do that substitution.  But, since you also seem to think that it should be possible to have that substitution, I want to give that a second shot.